test/browser/run-test.sh

```diff
@@ -4,25 +4,73 @@ set -eux
 # tests need cockpit's bots/ libraries and test infrastructure
 cd $SOURCE
 git init
+rm -f bots  # common local case: existing bots symlink
 make bots test/common
 
-# only install a subset to save time/space
-rm -f package-lock.json  # otherwise the command below installs *everything*, argh
-npm install chrome-remote-interface sizzle
+# support running from clean git tree
+if [ ! -d node_modules/chrome-remote-interface ]; then
+    npm install chrome-remote-interface sizzle
+fi
+
+# disable detection of affected tests; testing takes too long as there is no parallelization,
+# and TF machines are slow and brittle
+mv .git dot-git
 
 . /etc/os-release
 export TEST_OS="${ID}-${VERSION_ID/./-}"
+export TEST_AUDIT_NO_SELINUX=1
 # HACK: upstream does not yet know about rawhide
-if [ "$TEST_OS" = "fedora-34" ]; then
-    export TEST_OS=fedora-33
+if [ "$TEST_OS" = "fedora-35" ]; then
+    export TEST_OS=fedora-34
 fi
 
-export TEST_AUDIT_NO_SELINUX=1
+if [ "$ID" = "fedora" ]; then
+    # Testing Farm machines are really slow at some times of the day
+    export TEST_TIMEOUT_FACTOR=3
+fi
 
 EXCLUDES=""
 
+# triggers SELinux violation
+# See journal: SELinux is preventing /usr/libexec/qemu-kvm from open access on the file /var/lib/cockpittest/nfs_pool/nfs-volume-0.
+EXCLUDES="$EXCLUDES TestMachinesDisks.testAddDiskNFS"
+# FIXME: investigate
+EXCLUDES="$EXCLUDES TestMachinesNetworks.testNetworkSettings"
```

Comment:
This is "just" a flake -- it happens in our CI as well, e.g. [ubuntu-stable](https://logs-https-frontdoor.apps.ocp.ci.centos.org/logs/pull-16-20210325-050856-8a9e6389-ubuntu-stable/log.html) and [rhel-8-4](https://logs.cockpit-project.org/logs/pull-16-20210325-050856-8a9e6389-rhel-8-4/log.html)